src/ui/qgssourcefieldsproperties.ui

```diff
@@ -74,27 +94,23 @@
        </property>
       </widget>
      </item>
-     <item row="0" column="1">
-      <widget class="QToolButton" name="mDeleteAttributeButton">
-       <property name="toolTip">
-        <string>Delete field</string>
-       </property>
-       <property name="text">
-        <string/>
-       </property>
-       <property name="icon">
-        <iconset>
-         <normaloff>../../../../.designer/backup/.designer/xpm/delete_attribute.png</normaloff>../../../../.designer/backup/.designer/xpm/delete_attribute.png</iconset>
+     <item row="0" column="6">
+      <spacer name="horizontalSpacer_2">
+       <property name="orientation">
+        <enum>Qt::Horizontal</enum>
        </property>
-       <property name="shortcut">
-        <string>Ctrl+X</string>
+       <property name="sizeHint" stdset="0">
+        <size>
+         <width>40</width>
+         <height>20</height>
+        </size>
        </property>
-      </widget>
+      </spacer>
      </item>
      <item row="0" column="3">
-      <widget class="QToolButton" name="mCalculateFieldButton">
+      <widget class="QToolButton" name="mSaveLayerEditsButton">
        <property name="toolTip">
-        <string>Field calculator</string>
+        <string>Save Layer Edits</string>
```

Comment:
Can you also add the new button to the tabstop at the end of the file?

Reply:
what do you mean exactly?

Reply:
Add this button at line 127.

Reply:
done and done!